packages/core/src/components/forms/_controls.scss

```diff
@@ -197,7 +197,7 @@ $control-indicator-spacing: $pt-grid-size !default;
 
   input:focus ~ .#{$ns}-control-indicator {
     @include focus-outline();
-    outline: $blue3 solid 2px;
+    outline: $blue3 solid 2px; // remove this line for bp6 for a consistent focus outline style
```

Comment:
now that the contrast is bumped up by default, I think we can get rid of this override as part of a breaking change